tests/opentracer/test_tracer.py

```diff
@@ -2,179 +2,159 @@
 
```

Comment:
Note that changes to this file are just:

- using the fixtures defined in `tests/opentracer/utils.py`
- formatting using `black`